### PR TITLE
903143 - Prepare UG documentation for the Volume 3 features

### PR DESCRIPTION
**Pull request by ManjudhanasekaranSF4622**
Prepared UG document for the volume 3 feature.

- CloseButton Customization support for Syncfusion Editor controls
- SfButton Text Transform support
- Return Command support for Syncfusion Editor controls
- Support to Get the Signature points in SfSignaturePad

### CODE REVIEW

File: MAUI/Autocomplete/UI-Customization.md [hunk 1]
```diff
@@ -1212,3 +1212,88 @@ N> Dynamic changes to the `HorizontalTextAlignment` property may not be function
 The following image illustrates the result of the above code:
 
 ![TextAlignment](Images/UICustomization/TextAlignment.png)
+
+## Clear button customization
+
+The `ClearButtonPath` property is used to customize the appearance of the clear button.
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/UI-Customization.md:1218]: This is so simple, In UG we have to provide adequate info.

File: MAUI/Autocomplete/UI-Customization.md [hunk 2]
```diff
@@ -1212,3 +1212,88 @@ N> Dynamic changes to the `HorizontalTextAlignment` property may not be function
...
+        };
+
+        autoComplete.ClearButtonPath = path;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![ClearButtonPath](Images/UICustomization/clearbuttoncustomization.png)
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/UI-Customization.md:1256]: Include control name.

File: MAUI/Autocomplete/UI-Customization.md [hunk 3]
```diff
@@ -1212,3 +1212,88 @@ N> Dynamic changes to the `HorizontalTextAlignment` property may not be function
...
+
+## Command
+
+### Return Command and parameter
+
+`ReturnCommand` and `ReturnCommandParameter` is used to handle and customization behavior of pressing the Enter key. It allows user to define and control what should happen when presses the Enter key and to provide extra details that might be need to customize the action.
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/UI-Customization.md:1262]: Check grammatical errors.

Comment (PrithisIyyappan) [reply MAUI/Autocomplete/UI-Customization.md:1262]: Add links if possible.

File: MAUI/Autocomplete/UI-Customization.md (hunk 1 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md]: change updated

File: MAUI/Autocomplete/UI-Customization.md (hunk 2 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md]: included the control name

File: MAUI/Autocomplete/UI-Customization.md (hunk 3 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md]: changes updated

File: MAUI/Autocomplete/UI-Customization.md [hunk 4]
```diff
@@ -1240,3 +1240,80 @@ sfAutocomplete.ReturnType = ReturnType.Next;
...
+
+![.NET MAUI Autocomplete ClearButtonPath](Images/UICustomization/clearbuttoncustomization.png)
+
+## Command
+
+### Return Command and Return Command Parameter
+
+`ReturnCommand` and `ReturnCommandParameter` allows users to customize and handle action invoked by pressing the Enter key.
+
+- `ReturnCommand` : Specifies what should happent when a [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html)control is interacted (e.g., clicking a button or pressing Enter key in an entry field).
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/UI-Customization.md:1292]: ReturnCommand, of type [ICommand](https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.icommand), defines the command to be executed when the return key is pressed.

File: MAUI/Autocomplete/UI-Customization.md [hunk 5]
```diff
@@ -1240,3 +1240,80 @@ sfAutocomplete.ReturnType = ReturnType.Next;
...
+<editors:SfAutocomplete x:Name="autocomplete"
+                    ItemsSource="{Binding SocialMedia}"
+                    DisplayMemberPath="Name"
+                    TextMemberPath="Name"
+                    ReturnCommand="{Binding AlertCommand}"
+                    ReturnCommandParameter="Enter key is pressed"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/UI-Customization.md:1307]: Add xaml equivalent c# code

File: MAUI/Button/Customization.md [hunk 6]
```diff
@@ -124,6 +124,34 @@ button.VerticalTextAlignment = TextAlignment.Center;
 {% endhighlight %}
 {% endtabs %}
 
+## Text Transform
+
+Users can now customize the [`SfButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfButton.html) text using the `TextTransform` property. They can easily switch between uppercase, lowercase, none, or default styles.
```
Comment (Choza-rajan) [inline MAUI/Button/Customization.md:129]: Users can now customize the [`SfButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfButton.html) text casing using the `TextTransform` property. They can easily switch between uppercase, lowercase, none, or default casing.

File: MAUI/ComboBox/UI-Customization.md [hunk 7]
```diff
@@ -1402,3 +1402,81 @@ sfComboBox.ReturnType = ReturnType.Next;
 {% endtabs %}
 
 ![.NET MAUI ComboBox ReturnType](Images/UICustomization/ReturnType.png)
+
+## Clear button customization
+
+The `ClearButtonPath` property allows users to set the path for customizing the appearance of the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) clear Button.
```
Comment (Choza-rajan) [inline MAUI/ComboBox/UI-Customization.md:1408]: clear Button. recheck the casing

File: MAUI/Autocomplete/UI-Customization.md (hunk 5 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md:1323]: Added xaml equivalent c# code

File: MAUI/Autocomplete/UI-Customization.md (hunk 4 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md]: content updated

File: MAUI/Autocomplete/UI-Customization.md [hunk 8]
```diff
@@ -1240,3 +1240,80 @@ sfAutocomplete.ReturnType = ReturnType.Next;
...
+![.NET MAUI Autocomplete ClearButtonPath](Images/UICustomization/clearbuttoncustomization.png)
+
+## Command
+
+### Return Command and Return Command Parameter
+
+`ReturnCommand` and `ReturnCommandParameter` allows users to customize and handle action invoked by pressing the Enter key.
+
+- `ReturnCommand` : Specifies what should happent when a [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html)control is interacted (e.g., clicking a button or pressing Enter key in an entry field).
+- `ReturnCommandParameter` : Provides additional data needed for the action.
```
[the remark this answers is not in the document]
Comment (ManjudhanasekaranSF4622) [reply MAUI/Autocomplete/UI-Customization.md]: content updated

File: MAUI/Button/Customization.md (hunk 6 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/Button/Customization.md]: content updated

File: MAUI/ComboBox/UI-Customization.md (hunk 7 above)
Comment (ManjudhanasekaranSF4622) [reply MAUI/ComboBox/UI-Customization.md]: change updated

File: MAUI/Button/Customization.md [hunk 9]
```diff
@@ -124,6 +124,34 @@ button.VerticalTextAlignment = TextAlignment.Center;
...
+SfButton button = new SfButton();
+button.Text = "Submit";
+button.TextTransform = "Uppercase";
+button.HorizontalTextAlignment = TextAlignment.Center;
+button.VerticalTextAlignment = TextAlignment.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with texttransform](images/customization-images/Button_texttransform.png)
```
Comment (PrithisIyyappan) [inline MAUI/Button/Customization.md:153]: .NET MAUI Button Text Transform -- change like this

File: MAUI/SignaturePad/getting-started.md [hunk 10]
```diff
@@ -264,4 +264,37 @@ private void OnDrawCompleted(object? sender, EventArgs e)
 
 {% endtabs %}
 
+## Method
+
+### Get Signature Points
+
+Users can use the `GetSignaturePoints` method to retrieve the drawn signature points from the [`SfSignaturePad`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SignaturePad.SfSignaturePad.html) after the completion of the signature drawing.
```
[the remark this answers is not in the document]
Comment (ManjudhanasekaranSF4622) [reply MAUI/SignaturePad/getting-started.md]: @Choza-rajan sir, I have removed the words sir.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**